src/components/TaxonDetails/TaxonDetails.js

```diff
@@ -64,6 +68,7 @@ const TaxonDetails = ( ): Node => {
     logger.error( `Failed to retrieve taxon ${id}: ${error}` );
   }
   const taxon = remoteTaxon || localTaxon;
+  const taxonUrl = `${TAXON_URL}/${taxon.id}`;
```

Comment:
This line is throwing an error for me on Android.

<img width="426" alt="Screenshot 2023-12-18 at 2 46 18 PM" src="https://github.com/inaturalist/iNaturalistReactNative/assets/35536439/e9b34823-cf11-4f39-a02f-59626552c263">